packages/perseus/src/widgets/explanation/prompt-utils.ts

```diff
@@ -0,0 +1,22 @@
+import {WidgetType} from "../../prompt-types";
+
+import type explanation from "./explanation";
+import type {PropsFor} from "@khanacademy/wonder-blocks-core";
+
+export type ExplanationPromptJSON = {
+    type: WidgetType;
+    showPrompt: PropsFor<typeof explanation.widget>["showPrompt"];
```

Comment:
If we decide to not go with my suggestion of writing out explicit, segregated types for the prompt JSON API, I'd rather we just export `Props` or something from Explanation and using that instead of `PropsFor<typeof explanation.widget>`